api/bazel/repository_locations.bzl

```diff
@@ -32,9 +32,9 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Bazel build tools",
         project_desc = "Developer tools for working with Google's bazel buildtool.",
         project_url = "https://github.com/bazelbuild/buildtools",
-        version = "4.0.1",
-        sha256 = "c28eef4d30ba1a195c6837acf6c75a4034981f5b4002dda3c5aa6e48ce023cf1",
-        release_date = "2021-03-01",
+        version = "4.2.0",
+        sha256 = "d49976b0b1e81146d79072f10cabe6634afcd318b1bd86b0102d5967121c43c1",
+        release_date = "2021-09-06",
```

Comment:
Can we move to the latest `4.2.1`?

https://github.com/bazelbuild/buildtools/releases/tag/4.2.1

Reply:
yep, missed that, thanks, ill update

Reply:
@moderation it turns out that this is correct the current release is 4.2.0 - not sure why but 4.2.1 is only a tag not a release